accounts/models.py

```diff
@@ -236,6 +241,25 @@ class Payout(TimeStampedModel):
     note = models.CharField(max_length=500, blank=True)
```

Comment:
@autn help me understand what `note` is here? Because below we have code that says "filter by pk = self.note" which seems odd - how can the `note` be a PK below?

Reply:
@YPCrumble we are having 2 types of payout, and I used the `note` field to distinguish referral payout from the other payout. I think we should use another field instead of using the `note` field!